src/axom/core/Array.hpp

```diff
@@ -733,7 +736,7 @@ class Array : public ArrayBase<T, DIM, Array<T, DIM, SPACE>>
     resize({size}, true, &value);
   }
 
-  void resize(const StackArray<IndexType, DIM>& size, const T& value)
+  void reshape(const StackArray<IndexType, DIM>& size, const T& value)
```

Comment:
This is a public user facing function. This needs to be documented in the Release Notes.

Reply:
This is the work-around I need to back out.  If the alternative is still changes the public function, I'll document it.

Reply:
Is this issue #1082? I haven't gotten around to looking at it, sorry about that -- I'll see if I can push out a quick fix for you.

Reply:
#1082 should be resolved before this PR is merged to avoid changing `Array`'s API

Reply:
https://github.com/LLNL/axom/issues/1082 has been resolved and I've reverted the name change.